edx/edx-app-ios
# iOS - Profile edit view: Show error message in case location is failed to update.

### Description

[LEARNER-7367](https://openedx.atlassian.net/browse/LEARNER-7367)

Edit profile is breaking by updating country with staff user.

### How to test this PR

- Logged in with staff user
- Go into profile view 
- From top right corner go into edit mode.
- Open the location view and update the country.
- Error should show for staff user.
